### PR TITLE
done till 28 with corrections 

### PR DESCRIPTION
**Pull request by matmuret**

### CODE REVIEW

File: 28-PB-function-scope/Filippo.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    console.log(mult);
+}
+exponential(3, 5)
+
+//2
+let myFav = "Mango"
+const myFavoriteFruit = (myfav) => {
+    console.log(`My favourite fruit is ${myFav}`)
+}
+myFavoriteFruit(myfav)
```
Comment (martl) [inline 28-PB-function-scope/Filippo.js:19]: semicolon

File: 28-PB-function-scope/Filippo.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        mult = mult + ' ' + num1 ** i;
+    }
+    console.log(mult);
+}
+exponential(3, 5)
+
+//2
+let myFav = "Mango"
+const myFavoriteFruit = (myfav) => {
+    console.log(`My favourite fruit is ${myFav}`)
```
Comment (martl) [inline 28-PB-function-scope/Filippo.js:17]: overwrite myFav inside

File: 28-PB-function-scope/Filippo.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+//1
+const exponential = (num1, num2) => {
+    console.log(num1 + " and " + num2 + " are " + typeof (num1, num2) + "s");
+    let mult = '';
+    for (let i = 1; i <= num2; i++) {
```
Comment (martl) [inline 28-PB-function-scope/Filippo.js:7]: put the for inside a if, only do that if its a number!

File: 28-PB-function-scope/Filippo.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+"use strict";
+
+//1
+const exponential = (num1, num2) => {
+    console.log(num1 + " and " + num2 + " are " + typeof (num1, num2) + "s");
```
Comment (Ungoot) [inline 28-PB-function-scope/Filippo.js:5]: Include an if condition to check if they are numbers.

File: 28-PB-function-scope/Filippo.js [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    for (let i = 1; i <= num2; i++) {
+        mult = mult + ' ' + num1 ** i;
+    }
+    console.log(mult);
+}
+exponential(3, 5)
+
+//2
+let myFav = "Mango"
+const myFavoriteFruit = (myfav) => {
```
Comment (Ungoot) [inline 28-PB-function-scope/Filippo.js:16]: typo should be myFav

File: 28-PB-function-scope/Filippo.js (hunk 1 above)
Comment (Ungoot) [inline 28-PB-function-scope/Filippo.js:19]: typo should be myFav

File: 28-PB-function-scope/Filippo.js (hunk 2 above)
Comment (Ungoot) [inline 28-PB-function-scope/Filippo.js:17]: `Reassign the variable within the function`

File: 28-PB-function-scope/Filippo.js (hunk 1 above)
Comment (Ungoot) [inline 28-PB-function-scope/Filippo.js:19]: Typo. Should be myFav

**Pull request closed without merging**